client/src/components/TeamRow/TeamRow.tsx

```diff
@@ -9,15 +9,15 @@ interface Props {
 const TeamRow = ({ user }: Props) => {
   return (
     <div className="team-row-general drop-shadow-md">
-      <div className="flex flex-row w-[15%] justify-start ml-[15px]">
+      <div className="flex flex-row w-[20%] justify-start ml-[15px]">
```

Comment:
A recommendation, don't use so many tailwindcss for styling, we said it was just for specific things but you're using it for things that would be better to have in a css file

Reply:
I used tailwind here on all elements since they have different width, justify and margin values, but I noticed with this that I could just put the 'display' value on the CSS stylesheet so I did the changes.